src/index.js

```diff
@@ -67,7 +68,11 @@ class ReactToPrint extends React.Component {
       return;
     }
 
+    /** clear prev print window in here */
+    this.currentPrintWindow && this.removeWindow(this.currentPrintWindow);
```

Comment:
This will now run before we've actually printed anything, resulting in a blank page being printed.